--- origsrc/corebird-1.0.1/src/widgets/MediaVideoWidget.vala	2015-07-17 08:15:47.000000000 -0500
+++ src/corebird-1.0.1/src/widgets/MediaVideoWidget.vala	2015-08-28 17:07:13.861773600 -0500
@@ -34,7 +34,7 @@ class MediaVideoWidget : Gtk.Stack {
     drawing_area.realize.connect (realize_cb);
 #if VIDEO
     this.src  = Gst.ElementFactory.make ("playbin", "video");
-    this.sink = Gst.ElementFactory.make ("xvimagesink", "sink");
+    this.sink = Gst.ElementFactory.make ("ximagesink", "sink");
     this.src.set ("video-sink", sink, null);
     var bus = src.get_bus ();
     bus.set_sync_handler (bus_sync_handler);
